keras_cv/utils/fill_utils.py

```diff
@@ -10,42 +10,80 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.grep -q Copyright $i
+# limitations under the License.
 import tensorflow as tf
 
+from keras_cv.utils import bbox
 
-def fill_rectangle(
-    image, center_width, center_height, half_width, half_height, fill=None
-):
-    """Fill a rectangle in a given image using the value provided in replace.
+
+def rectangle_masks(mask_shape, corners):
+    """Computes positional masks of rectangles in images
 
     Args:
-        image: the starting image to fill the rectangle on.
-        center_width: the X center of the rectangle to fill
-        center_height: the Y center of the rectangle to fill
-        half_width: 1/2 the width of the resulting rectangle
-        half_height: 1/2 the height of the resulting rectangle
-        fill: A tensor with same shape as image. Values at rectangle
-         position are used as fill.
+        mask_shape: shape of the masks as [batch_size, height, width].
```

Comment:
I think its better to have shape as [batch, width, height] (usually x/y and width/height)

Reply:
solved in #98